docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -0,0 +1,95 @@
+# Training models with GPU support
```

Comment:
No mention of how to install CUDA / check that CUDA is available / check that AutoGluon detects the GPU?

Reply:
We should not bring CUDA installation topic into our docs: it's changing over time and across operation systems. I added a note that GPU traning requires CUDA installed and the official documentation [1] should be followed.

[1] https://docs.nvidia.com/cuda/